2022-01-05 00:01:43,010 INFO client.py "Имя клиента" test3
2022-01-05 00:01:43,011 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:01:43,061 INFO client.py "Имя клиента" test1
2022-01-05 00:01:43,062 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:01:43,066 INFO client.py "Имя клиента" test2
2022-01-05 00:01:43,066 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:01:43,084 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,085 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-05 00:01:43,089 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test3
2022-01-05 00:01:43,093 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,095 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=432, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60346), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641330103.0894876, 'USER': {'ACCOUNT_NAME': 'test3'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,106 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,106 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-05 00:01:43,113 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,114 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-05 00:01:43,119 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test1
2022-01-05 00:01:43,121 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,123 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=432, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60346), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,123 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:01:43,123 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=452, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60347), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641330103.1194904, 'USER': {'ACCOUNT_NAME': 'test1'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,124 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,126 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test2
2022-01-05 00:01:43,128 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,130 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=452, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60347), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,130 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:01:43,132 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,137 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60348), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641330103.1264913, 'USER': {'ACCOUNT_NAME': 'test2'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,142 DEBUG client.py Запущены процессы
2022-01-05 00:01:43,146 DEBUG client.py Запущены процессы
2022-01-05 00:01:43,643 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=312, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60348), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,643 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:01:43,644 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:01:43,647 DEBUG client.py Запущены процессы
2022-01-05 00:01:56,351 DEBUG client.py Сформирован словарь сообщения: {'ACTION': 'MESSAGE', 'SENDER': 'test3', 'DESTINATION': 'test1', 'TIME': 1641330116.3504837, 'MESSAGE_TEXT': 'fsdfsdfsdfsdfsdf'}
2022-01-05 00:01:56,353 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=432, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60346), raddr=('127.0.0.1', 7777)>, {'ACTION': 'MESSAGE', 'SENDER': 'test3', 'DESTINATION': 'test1', 'TIME': 1641330116.3504837, 'MESSAGE_TEXT': 'fsdfsdfsdfsdfsdf'}), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции create_message
2022-01-05 00:01:56,354 INFO client.py Отправлено сообщение для пользователя test1
2022-01-05 00:01:56,355 DEBUG client.py Была вызвана функция create_message
 c параметрами (<User_interface(Thread-2, started daemon 10376)>, <socket.socket fd=432, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60346), raddr=('127.0.0.1', 7777)>, 'test3'), {}.Вызов из модуля __main__.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:01:56,406 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=452, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60347), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:01:56,407 INFO client.py Получено сообщение от пользователя test3:
fsdfsdfsdfsdfsdf
2022-01-05 00:46:49,927 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:46:49,957 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:46:49,957 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: default
2022-01-05 00:46:52,006 ERROR client.py [WinError 10061] Подключение не установлено, т.к. конечный компьютер отверг запрос на подключение
2022-01-05 00:47:03,903 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:47:03,937 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:03,937 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: default
2022-01-05 00:47:03,938 DEBUG client.py Сформировано PRESENCE сообщение для пользователя default
2022-01-05 00:47:03,940 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('default',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:03,942 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60391), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332823.9385035, 'USER': {'ACCOUNT_NAME': 'default'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:04,446 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60391), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:04,446 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:47:04,447 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:04,449 DEBUG client.py Запущены процессы
2022-01-05 00:47:08,607 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60391), raddr=('127.0.0.1', 7777)>, {'ACTION': 'EXIT', 'TIME': 1641332828.6044967, 'SENDER': 'default'}), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:47:08,607 INFO client.py Завершение работы по команде пользователя.
2022-01-05 00:47:09,040 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=356, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60391), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:47:09,040 ERROR client.py Получено некорректное сообщение с сервера: {'RESPONSE': 400, 'ERROR': 'Запрос некорректен.'}
2022-01-05 00:47:09,481 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-05 00:47:19,626 INFO client.py "Имя клиента" test1
2022-01-05 00:47:19,627 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:47:19,627 INFO client.py "Имя клиента" test2
2022-01-05 00:47:19,628 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:47:19,689 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,689 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-05 00:47:19,694 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test1
2022-01-05 00:47:19,696 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,696 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-05 00:47:19,696 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,700 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60392), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332839.6944978, 'USER': {'ACCOUNT_NAME': 'test1'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,701 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test2
2022-01-05 00:47:19,703 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,706 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60392), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
tings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,706 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:47:19,708 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,718 DEBUG client.py Запущены процессы
2022-01-05 00:47:19,745 INFO client.py "Имя клиента" test3
2022-01-05 00:47:19,746 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:47:19,819 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,819 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-05 00:47:19,824 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test3
2022-01-05 00:47:19,829 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=448, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60393), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,829 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:47:19,831 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,834 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,838 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=348, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60394), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332839.824499, 'USER': {'ACCOUNT_NAME': 'test3'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:19,843 DEBUG client.py Запущены процессы
2022-01-05 00:47:20,351 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=348, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60394), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:20,351 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:47:20,353 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:47:20,355 DEBUG client.py Запущены процессы
2022-01-05 00:47:39,233 DEBUG client.py Сформирован словарь сообщения: {'ACTION': 'MESSAGE', 'SENDER': 'test1', 'DESTINATION': 'test3', 'TIME': 1641332859.2334952, 'MESSAGE_TEXT': 'dsfsdf'}
2022-01-05 00:47:39,237 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60392), raddr=('127.0.0.1', 7777)>, {'ACTION': 'MESSAGE', 'SENDER': 'test1', 'DESTINATION': 'test3', 'TIME': 1641332859.2334952, 'MESSAGE_TEXT': 'dsfsdf'}), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции create_message
2022-01-05 00:47:39,237 INFO client.py Отправлено сообщение для пользователя test3
2022-01-05 00:47:39,239 DEBUG client.py Была вызвана функция create_message
 c параметрами (<Interface(Thread-2, started daemon 10488)>, <socket.socket fd=388, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60392), raddr=('127.0.0.1', 7777)>, 'test1'), {}.Вызов из модуля __main__.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:47:39,249 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=348, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60394), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции self._bootstrap_inner().
 Вызов из функции run
2022-01-05 00:47:39,250 INFO client.py Получено сообщение от пользователя test1:
dsfsdf
2022-01-05 00:48:25,946 INFO client.py "Имя клиента" test2
2022-01-05 00:48:25,947 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:25,951 INFO client.py "Имя клиента" test3
2022-01-05 00:48:25,951 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:25,986 INFO client.py "Имя клиента" test1
2022-01-05 00:48:25,986 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:26,004 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,004 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-05 00:48:26,006 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test3
2022-01-05 00:48:26,014 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,014 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-05 00:48:26,019 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,020 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test2
2022-01-05 00:48:26,022 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,024 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60396), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332906.0204988, 'USER': {'ACCOUNT_NAME': 'test2'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,025 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60395), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332906.0064983, 'USER': {'ACCOUNT_NAME': 'test3'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,027 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60395), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,029 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'}
2022-01-05 00:48:26,031 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,035 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,035 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-05 00:48:26,035 ERROR client.py Критическая ошибка
2022-01-05 00:48:26,036 DEBUG client.py Запущены процессы
2022-01-05 00:48:26,038 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test1
2022-01-05 00:48:26,041 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,043 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=428, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60396), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,043 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'}
2022-01-05 00:48:26,045 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,047 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=420, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60397), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332906.0384986, 'USER': {'ACCOUNT_NAME': 'test1'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,048 ERROR client.py Критическая ошибка
2022-01-05 00:48:26,049 DEBUG client.py Запущены процессы
2022-01-05 00:48:26,556 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=420, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60397), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,556 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'}
2022-01-05 00:48:26,559 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 400, 'ERROR': 'Имя пользователя уже занято.'},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:26,561 ERROR client.py Критическая ошибка
2022-01-05 00:48:26,561 DEBUG client.py Запущены процессы
2022-01-05 00:48:27,049 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-05 00:48:27,065 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-05 00:48:27,575 DEBUG client.py Была вызвана функция main
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции <module>
2022-01-05 00:48:37,672 INFO client.py "Имя клиента" test1
2022-01-05 00:48:37,673 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:37,700 INFO client.py "Имя клиента" test2
2022-01-05 00:48:37,700 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:37,721 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,721 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test1
2022-01-05 00:48:37,725 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test1
2022-01-05 00:48:37,728 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test1',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,730 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60398), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332917.7254996, 'USER': {'ACCOUNT_NAME': 'test1'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,748 INFO client.py "Имя клиента" test3
2022-01-05 00:48:37,748 INFO client.py Установлен стандартный порт и адрес
2022-01-05 00:48:37,750 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,750 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test2
2022-01-05 00:48:37,756 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test2
2022-01-05 00:48:37,758 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test2',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,760 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=300, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60398), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,760 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=296, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60399), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332917.7564983, 'USER': {'ACCOUNT_NAME': 'test2'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,760 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:48:37,761 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,765 DEBUG client.py Запущены процессы
2022-01-05 00:48:37,796 DEBUG client.py Была вызвана функция get_args
 c параметрами (), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,797 INFO client.py Запущен клиент с парамертами: адрес сервера: 127.0.0.1, 
порт: 7777, имя пользователя: test3
2022-01-05 00:48:37,799 DEBUG client.py Сформировано PRESENCE сообщение для пользователя test3
2022-01-05 00:48:37,801 DEBUG client.py Была вызвана функция create_presence
 c параметрами ('test3',), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,803 DEBUG client.py Была вызвана функция send_message
 c параметрами (<socket.socket fd=456, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60400), raddr=('127.0.0.1', 7777)>, {'ACTION': 'PRESENCE', 'TIME': 1641332917.7994962, 'USER': {'ACCOUNT_NAME': 'test3'}}), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,803 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=296, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60399), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,804 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:48:37,805 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:37,808 DEBUG client.py Запущены процессы
2022-01-05 00:48:38,311 DEBUG client.py Была вызвана функция get_message
 c параметрами (<socket.socket fd=456, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=0, laddr=('127.0.0.1', 60400), raddr=('127.0.0.1', 7777)>,), {}.Вызов из модуля common.settings.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:38,311 DEBUG client.py Разбор приветственного сообщения от сервера: {'RESPONSE': 200}
2022-01-05 00:48:38,313 DEBUG client.py Была вызвана функция process_response_ans
 c параметрами ({'RESPONSE': 200},), {}.Вызов из модуля __main__.
 Вызов изфункции main().
 Вызов из функции main
2022-01-05 00:48:38,316 DEBUG client.py Запущены процессы
